sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -532,72 +532,56 @@ poller = text_analytics_client.begin_analyze_actions(
 )
 
 # returns multiple actions results in the same order as the inputted actions
-result = poller.result()
-
-first_action_result = next(result)
-print("Results of Entities Recognition action:")
-docs = [doc for doc in first_action_result.document_results if not doc.is_error]
-
-for idx, doc in enumerate(docs):
-    print("\nDocument text: {}".format(documents[idx]))
-    for entity in doc.entities:
-        print("Entity: {}".format(entity.text))
-        print("...Category: {}".format(entity.category))
-        print("...Confidence Score: {}".format(entity.confidence_score))
-        print("...Offset: {}".format(entity.offset))
-    print("------------------------------------------")
-
-second_action_result = next(result)
-print("Results of PII Entities Recognition action:")
-docs = [doc for doc in second_action_result.document_results if not doc.is_error]
-
-for idx, doc in enumerate(docs):
-    print("Document text: {}".format(documents[idx]))
-    for entity in doc.entities:
-        print("Entity: {}".format(entity.text))
-        print("Category: {}".format(entity.category))
-        print("Confidence Score: {}\n".format(entity.confidence_score))
-    print("------------------------------------------")
-
-third_action_result = next(result)
-print("Results of Key Phrase Extraction action:")
-docs = [doc for doc in third_action_result.document_results if not doc.is_error]
-
-for idx, doc in enumerate(docs):
-    print("Document text: {}\n".format(documents[idx]))
-    print("Key Phrases: {}\n".format(doc.key_phrases))
-    print("------------------------------------------")
-
-fourth_action_result = next(result)
-print("Results of Linked Entities Recognition action:")
-docs = [doc for doc in fourth_action_result.document_results if not doc.is_error]
-
-for idx, doc in enumerate(docs):
-    print("Document text: {}\n".format(documents[idx]))
-    for linked_entity in doc.entities:
-        print("Entity name: {}".format(linked_entity.name))
-        print("...Data source: {}".format(linked_entity.data_source))
-        print("...Data source language: {}".format(linked_entity.language))
-        print("...Data source entity ID: {}".format(linked_entity.data_source_entity_id))
-        print("...Data source URL: {}".format(linked_entity.url))
-        print("...Document matches:")
+pages = poller.result()
+
+for doc, document_results in zip(documents, pages):
+    print("\nDocument text: {}".format(doc))
+    recognize_entities_result = document_results[0]
+    assert not recognize_entities_result.is_error
```

Comment:
This is probably not a good use of an `assert`. Any document could have failed for a transient reason, so you are not really in a position to say that this didn't fail. I would print a "failed" message instead...

Reply:
Actually, scratch that, this is in the `readme.md`. It is a bit long for the readme already. I don't think we need to show all types of actions in the readme example inline. It definitely won't scale as new action types comes along. Can we limit this to two?